readthedocs/config/config.py

```diff
@@ -670,6 +671,19 @@ class BuildConfigV2(BuildConfigBase):
         'dirhtml': 'sphinx_htmldir',
         'singlehtml': 'sphinx_singlehtml',
     }
+    valid_os = ['ubuntu-20.04']
+    valid_languages = {
+        'python': [
+            '2.7',
+            '3', '3.8', '3.9', '3.10',
```

Comment:
and, are we allowing to specify the patch form? or just the minor version? are we including 3.7 and 3.6?

Reply:
I removed the patch number on purpose. As a user, there is no difference between `3.6.0` and `3.6.1`. However, in the backend we will need to have cached both patch versions to make the builds run fast; and that's what I'm trying to avoid.